CODE_OF_CONDUCT.md

```diff
@@ -1,9 +0,0 @@
-# Microsoft Open Source Code of Conduct
```

Comment:
Don't we need these anymore? The benefit of this file is that it shows up on the top of the repo:
![image](https://github.com/Azure/enterprise-azureai/assets/1424596/043f0512-07e5-4c7d-815f-7c85709e4577)

Reply:
No, CoC will remain, License and security will still be there.

Reply:
If I look at your PR branch, the code of conduct is not visible anymore (https://github.com/pascalvanderheiden/ph-enterprise-azureai/tree/main). Which makes sense, because it is removed in this PR. :)

Reply:
Strange, because we didn't have it in the beginning and tab CoC already showed. Ok, we'll leave it in.